addons/stock/views/help_message_template.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="help_message_template" name="Help Message" groups="base.group_user">
```

Comment:
Don't create a new file for help message template. Put them in their related xml views.